### PR TITLE
feat: databendWriter support replace mode

### PR DESCRIPTION
**Pull request by hantmac**
- upgrade databend-jdbc to 0.1.0
- support replace mode for databend

### CODE REVIEW

File: databendwriter/src/main/java/com/alibaba/datax/plugin/writer/databendwriter/util/DatabendWriterUtil.java [hunk 1]
```diff
@@ -1,40 +1,61 @@
...
-
-        StringBuilder writeDataSqlTemplate = new StringBuilder();
-        writeDataSqlTemplate.append("INSERT INTO %s");
-        StringJoiner columnString = new StringJoiner(",");
-
-        for (String column : columns) {
-            columnString.add(column);
+        LOG.info("write mode is {}", writeMode);
+        if (writeMode.toLowerCase().contains("replace")) {
+            // for databend if you want to use replace mode, the writeMode should be:  "writeMode": "replace (userid)"
```
Comment (dingxiaobo) [inline databendwriter/src/main/java/com/alibaba/datax/plugin/writer/databendwriter/util/DatabendWriterUtil.java]: `"writeMode": "replace (userid)"` 不太符合datax的写法，把这两个分开吧：

``` json
{
"writeMode": "replace"
"conflictColumn" : ["userid", "..."]
}
```

Comment (dingxiaobo) [reply databendwriter/src/main/java/com/alibaba/datax/plugin/writer/databendwriter/util/DatabendWriterUtil.java]: 在 `DataX/databendwriter/doc/databendwriter*.md` 里也说明一下replace 怎么使用吧

**Review by dingxiaobo — CHANGES_REQUESTED** [review]

File: databendwriter/src/main/java/com/alibaba/datax/plugin/writer/databendwriter/util/DatabendWriterUtil.java [hunk 2]
```diff
@@ -1,40 +1,63 @@
...
-        writeDataSqlTemplate.append("INSERT INTO %s");
-        StringJoiner columnString = new StringJoiner(",");
-
-        for (String column : columns) {
-            columnString.add(column);
+        LOG.info("write mode is {}", writeMode);
+        if (writeMode.toLowerCase().contains("replace")) {
+            if (onConflictColumns == null || onConflictColumns.size() == 0) {
+                LOG.error("Replace mode must has onConflictColumn conf");
+                return;
```
Comment (dingxiaobo) [inline databendwriter/src/main/java/com/alibaba/datax/plugin/writer/databendwriter/util/DatabendWriterUtil.java]: 看起来抛异常，让任务失败更合适

**Review by dingxiaobo — CHANGES_REQUESTED** [review]

**Comment by hantmac** [comment]
HI @dingxiaobo , would u like to help me review again? Thanks very much !

**Pull request merged**